duckduckgo/macos-browser
# new tab page polish

Task/Issue URL: https://app.asana.com/0/392891325557410/1199666163277445
Tech Design URL:
CC:

**Description**:

Add logo and search field to new tab page.

This contains slightly experimental animated SVG which I'm putting through product review.  If that's approved then we might consider using lottie instead (though this approach seems to work fine).

**Steps to test this PR**:
1. Open a new tab
1. Check the address bar gets first responder (not the search field)
1. Check the search field performs like the address bar - with suggestions and that the UI looks correct
1. Check dark/light theme applied correctly and dynamically

***Dax logo***:
1. Will show a random animation with a delay of between 30 and 120 seconds
1. Can click on it to show a random animation

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
